src/coreclr/pal/src/include/pal/dbgmsg.h

```diff
@@ -494,7 +494,7 @@ if new_level is -1, the nesting level will not be modified
 --*/
 int DBG_change_entrylevel(int new_level);
 
-#ifdef __APPLE__
+#ifdef TARGET_OSX
```

Comment:
Is `TARGET_OSX` a subset of `TARGET_APPLE`?

Reply:
Yes, it is.